wdl/MasterVcfQc.wdl

```diff
@@ -131,13 +134,15 @@ workflow MasterVcfQc {
       runtime_override_tar_shard_vid_lists=runtime_override_tar_shard_vid_lists,
   }
 
+  Int max_gq_ = if (defined(max_gq)) then select_first([max_gq]) else 99
```

Comment:
```suggestion
  Int max_gq_ = select_first([max_gq, 99])
```

Reply:
Should it be 99 or 999?

Reply:
I think it should be 99 since the most common place to run MasterVcfQc is after CleanVcf when the GQ will be rescaled, and it is easy to override in another scenario